encoding.bs

```diff
@@ -1110,16 +1110,20 @@ interface mixin TextDecoderCommon {
 </pre>
 
 <p>The {{TextDecoderCommon}} interface mixin defines common attributes that are shared between
-{{TextDecoder}} and {{TextDecoderStream}} objects. These objects have an associated
-<dfn id=textdecoder-encoding for=TextDecoderCommon>encoding</dfn>,
-<dfn id=textdecoder-ignore-bom-flag for=TextDecoderCommon>ignore BOM</dfn> (initially false),
-<dfn id=textdecoder-bom-seen-flag for=TextDecoderCommon>BOM seen</dfn> (initially false), and
-<dfn id=textdecoder-error-mode for=TextDecoderCommon>error mode</dfn> (initially
-"<code>replacement</code>").
+{{TextDecoder}} and {{TextDecoderStream}} objects. These objects have an associated:
+
+<ul class=brief>
```

Comment:
This really feels like a definition list to me, but I'm not sure if that'd end up looking right.